Integrations/Analytics/Adobe Analytics/List Variable Integration/integration.js

```diff
@@ -1,41 +1,62 @@
-// This extension integrates Optimizely w/ Adobe via a pre-defined list variable. Set the list variable on line 3 below.
-var decisionString = window.optimizely.get('state').getDecisionString({campaignId: campaignId, shouldCleanString: true});
+var decisionString = window.optimizely.get('state').getDecisionString({
+  campaignId: campaignId,
+  shouldCleanString: true
+});
 var redirectInfo = window.optimizely.get('state').getRedirectInfo();
-var list = "list1";
```

Comment:
@cooperreid-optimizely what's the rational from moving away from a hard-coded list variable key? The reason I put this in was because customers didn't want to modify the list variable used every time so the integration would genuinely be "one click". Just wanted to confirm there's a reason before I merge in.